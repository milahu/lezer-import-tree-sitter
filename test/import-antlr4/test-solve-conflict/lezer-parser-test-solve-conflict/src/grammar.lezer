@top prog {
  expr
}

expr {
  expr (
    '*' |
    '/'
  ) expr |
  expr (
    '+' |
    '-'
  ) expr |
  INT |
  '(' expr ')'
}

@tokens {

  NEWLINE {
    $[\r\n]+
  }

  INT {
    $[0-9]+
  }
}

@skip {
  NEWLINE
}
