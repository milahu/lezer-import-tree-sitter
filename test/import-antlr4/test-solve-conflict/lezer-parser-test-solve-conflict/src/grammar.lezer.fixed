@precedence {
  prec1 @left
}

@top prog {
  expr
}

expr {
  expr (
    !prec1 '*' |
    '/'
  ) expr |
  expr (
    '+' |
    '-'
  ) expr |
  INT |
  '(' expr ')'
}

@tokens {

  NEWLINE {
    $[\r\n]+
  }

  INT {
    $[0-9]+
  }
}

@skip {
  NEWLINE
}
