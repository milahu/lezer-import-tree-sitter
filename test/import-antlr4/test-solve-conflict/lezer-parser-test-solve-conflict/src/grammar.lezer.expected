@precedence {
  muldiv @left,
  addsub @left
}

@top prog {
  expr
}

expr {
  expr !muldiv (
    '*' |
    '/'
  ) expr |
  expr !addsub (
    '+' |
    '-'
  ) expr |
  INT |
  '(' expr ')'
}

@tokens {

  NEWLINE {
    $[\r\n]+
  }

  INT {
    $[0-9]+
  }
}

@skip {
  NEWLINE
}
